src/main/java/com/urbancode/ds/jenkins/plugins/urbandeploypublisher/UrbanDeployPublisher.java

```diff
@@ -434,42 +488,68 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
                     resolvedVersion,
                     listener);
 
-            listener.getLogger().println("Deployment request created with id: " + requestId);
-            listener.getLogger().println("Deployment of application request " + requestId
-                    + " of application " + resolvedDeployApp + " is running.");
+            listener.getCreateProcess().println("Deployment request created with id: " + requestId);
```

Comment:
What is this??? Listener doesn't have a getCreateProcess function.

Reply:
Some shortcut keys messed it up.